05_stringer/n0npax/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output. Expected: %q - Actual: %q", expected, actual)
+	}
+}
+
+func TestStringify(t *testing.T) {
+	testCases := map[string]struct {
+		input    IPAddr
+		expected string
+	}{
+		"localhost":            {input: IPAddr{127, 0, 0, 1}, expected: "127.0.0.1"},
+		"google primary dns":   {input: IPAddr{8, 8, 8, 8}, expected: "8.8.8.8"},
+		"google secondary dns": {input: IPAddr{8, 8, 4, 4}, expected: "8.8.4.4"},
+		"CloudFare":            {input: IPAddr{1, 1, 1, 1}, expected: "1.1.1.1"},
+		"mask 24":              {input: IPAddr{255, 255, 255, 0}, expected: "255.255.255.0"},
+		"mask 32":              {input: IPAddr{255, 255, 255, 255}, expected: "255.255.255.255"},
+		"mask 0":               {input: IPAddr{0, 0, 0, 0}, expected: "0.0.0.0"},
+		"anz":                  {input: IPAddr{202, 2, 59, 40}, expected: "202.2.59.40"},
+		"empty":                {input: IPAddr{}, expected: "0.0.0.0"},
+		"half-empty":           {input: IPAddr{10, 10}, expected: "10.10.0.0"},
+	}
+	for name, test := range testCases {
+		input, expected := test.input, test.expected
+		t.Run(name, func(t *testing.T) {
+			mt := assert.New(t)
```

Comment:
I'm not really sure what `mt` is supposed to stand for here. It is super minor so I won't block you from merging, but I would suggest more descriptive. When I read something like 

```go
assert := assert.New()
assert.Equalf(expected, actual, "Testcase %s failed")
```
It is a lot clearer to me. Although to do this you would have to provide an alias for the `testify/assert` package, something like below:
```go
import assertion "github.com/stretchr/testify/assert"

assert := assertion.New()
assert.Equalf(expected, actual, "Testcase %s failed")
```

Reply:
In other PR I was suggested to provide a similar construct. The aliased `assert` is much more readable. Thanks for pointing that.